src/targeting.js

```diff
@@ -49,25 +49,41 @@ export function newTargeting(auctionManager) {
         });
       });
     });
+
+    targeting = targeting.map(targeting => {
+      return {
+        [Object.keys(targeting)[0]]: targeting[Object.keys(targeting)[0]]
+          .map(target => {
+            return {
+              [Object.keys(target)[0]]: target[Object.keys(target)[0]].join(', ')
+            };
+          }).reduce((p, c) => Object.assign(c, p), {})
+      };
+    }).reduce(function (accumulator, targeting) {
+      var key = Object.keys(targeting)[0];
+      accumulator[key] = Object.assign({}, accumulator[key], targeting[key]);
+      return accumulator;
+    }, {});
+
     return targeting;
   };
 
   targeting.setTargeting = function(targetingConfig) {
```

Comment:
this should be setTargetingForDFP.

Reply:
After discussion it was decided to name this function `setTargetingForGPT` to make it consistent with public api `$$PREBID_GLOBAL$$.setTargetingForGPTAsync`